extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt

```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0"?>
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
```

Comment:
what does this file do?

Reply:
By being there it prevents main transformation file from applying to scm host.

Reply:
Add an XML comment to that effect